tests/check/check_h26xsigned_sign.c

```diff
@@ -129,6 +129,12 @@ START_TEST(api_inputs)
   ck_assert_int_eq(sv_rc, SV_INVALID_PARAMETER);
   sv_rc = signed_video_set_recurrence_interval(sv, 0);
   ck_assert_int_eq(sv_rc, SV_NOT_SUPPORTED);
+  sv_rc = signed_video_set_recurrence_interval_frames(sv, 1);
+  ck_assert_int_eq(sv_rc, SV_OK);
```

Comment:
Put the correct call after the incorrect ones. If we do not allow a parameter to change, the order is important.